whois-api/src/main/java/net/ripe/db/whois/api/mail/EmailMessageInfo.java

```diff
@@ -12,7 +13,7 @@ public String messageId() {
 
     @Override
     @Nullable
-    public String emailAddress() {
-        return emailAddress;
+    public List<String> emailAddresses() {
```

Comment:
You do not need a getter method for a record?

Reply:
you, with a record you can get to that value without a get!

recordClass.propertyName

Reply:
yes remove the getters then ?

Reply:
....true

Reply:
Done